backend/tarsy/database/init_db.py

```
@@ -12,6 +12,10 @@
 
 from tarsy.config.settings import get_settings
 
+# Import all SQLModel table classes to ensure they are registered for schema creation
+from tarsy.models.db_models import AlertSession, StageExecution
```

Ruff (0.12.2)

`tarsy.models.db_models.AlertSession` imported but unused

Remove unused import

(F401)